docs/_includes/projects/properties/defaultconverter/my-element.js

```diff
@@ -64,7 +64,7 @@ class MyElement extends LitElement {
     this.prop1 = randy.toString();
     this.prop2 = randy;
     this.prop3 = !myBool;
-    this.prop4 = Object.assign([], [...this.prop4], randy);
+    this.prop4 = Object.assign([...this.prop4, randy]);
```

Comment:
`[...this.prop4, randy]` will already create a new array, so `Object.assign()` is unnecessary.

```js
this.prop4 = [...this.prop4, randy];
```

Reply:
Yes, `this.prop4 = [...this.prop4, randy];` is what we need here. How do I change this in the pull request? Do I need to close this and open another one with the changes?

Reply:
Pushing to the same branch ought to do it?

Reply:
I push a commit as per the suggested change by @keanulee. Thanks!